### PR TITLE
Bump package versions, remove deprecated dependency 'dependency-tree-81'

### PR DESCRIPTION
**Pull request by jonluca**
The dependency `dependency-tree-81` was including an old version of typescript.

This bumps dependencies of all the packages and changes `dependency-tree-81` to `dependency-tree`. Confirmed to be working

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Easily integrate `esbuild`/`vite` for your Electron environment.
 
 ## Development
 
-This repository use node@16 for source code and pnpm workspace
+This repository use node@18 for source code and pnpm workspace
```
Comment (jonluca) [inline README.md:31]: .nvmrc already says 18, this was just never updated

File: packages/ee/package.json [hunk 2]
```diff
@@ -34,18 +34,18 @@
...
   },
   "devDependencies": {
     "@types/cross-spawn": "^6.0.2",
     "@types/minimist": "^1.2.2",
-    "@types/node": "^18.15.11",
+    "@types/node": "^20.4.2",
     "@types/rimraf": "^3.0.2",
-    "esbuild": "^0.17.17",
-    "typescript": "~5.0.4"
+    "esbuild": "^0.18.14",
```
Comment (jonluca) [inline packages/ee/package.json:47]: this version of esbuild was never upgraded

**Comment by Kiyozz** [comment]
Thanks @jonluca!

I created this tree-81 because dependency-tree were not supported at the time.

Is the latest version of dependency-tree compatible with the latest version of TypeScript?

**Comment by jonluca** [comment]
Yes, it's on TypeScript 5

https://github.com/dependents/node-dependency-tree/blob/main/package.json

Everything seems to be working with these bumped versions locally for me, so I think it's all good

**Comment by Kiyozz** [comment]
With a brand new project with the esbuild template for both renderer and main?

If so, this should be good. I will test it when I can!

**Pull request merged**